eigenda/certificate.go

```diff
@@ -102,27 +102,27 @@ func (e *EigenDAV1Cert) ToDisperserBlobInfo() (*disperser.BlobInfo, error) {
 	xBytes := e.BlobHeader.Commitment.X.Bytes()
 	yBytes := e.BlobHeader.Commitment.Y.Bytes()
 
-	// Remove 0 byte padding (if applicable)
+	// Remove 0x0 byte padding or add (if applicable)
```

Comment:
"or add" is unclear here. Guess you meant "and pad"?

Reply:
yup